scripts/user_setup.sh

```diff
@@ -0,0 +1,50 @@
+#!/usr/bin/bash
+
+# bash script to setup user account for htr2hpc pilot integration
+# - adds ssh key to authorized keys
+# - conda env setup
+# - create htr2hpc working directory in scratch
+
+echo "Setting up your account for htr2hpc ...."
```

Comment:
```suggestion
echo "Setting up your account for htr2hpc ...."
echo "This process may take at least five minutes. Please do not exit until the process completes."
```

Suggesting just that little extra note for the user -- otherwise I think we are good to merge!